lib/DxilContainer/DxilContainerAssembler.cpp

```diff
@@ -1657,9 +1657,25 @@ class DxilRDATWriter : public DxilPartWriter {
     }
 
     if (props.Node.LaunchType == NodeLaunchType::Mesh) {
+      const DxilEntrySignature &sig = entryProps.sig;
       nAttrib = {};
       nAttrib.AttribKind = (uint32_t)RDAT::NodeFuncAttribKind::MeshShaderInfo;
       RDAT::MSInfo info = {};
+      uint32_t shaderFlags = 0;
+      info.SigOutputElements = AddSigElements(sig.OutputSignature, shaderFlags);
```

Comment:
An aside: I found this pretty confusing until I realized that "shaderFlags" is a write-only out param that's taken as a non-const reference.  Totally beyond the scope of this PR, just wanted to have my say somewhere!